tools/bump_ocp_releases.py

```diff
@@ -287,13 +261,13 @@ def update_release_images_json(default_release_images_json, updates_made, update
                                                                                             latest_ocp_release_version)
 
     if updates_made:
-        with open(os.path.join(ASSISTED_SERVICE_CLONE_DIR, DEFAULT_RELEASE_IMAGES_FILE), 'w') as outfile:
+        with (clone_dir / DEFAULT_RELEASE_IMAGES_FILE).open("w") as outfile:
```

Comment:
Why not use `clone_dir.joinpath(DEFAULT_RELEASE_IMAGES_FILE)`?

Reply:
great idea!
fixed